test/mocks/overrides/override_user_centauri_points_view_model.dart

```diff
@@ -0,0 +1,47 @@
+import "package:flutter/foundation.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/models/database/user/user_id_firestore.dart";
+import "package:proxima/viewmodels/user_centauri_points_view_model.dart";
+
+/// A mock implementation of the [UserCentauriPointsViewModel] class.
+class MockUserAvatarCentauriPointsViewModel
+    extends FamilyAsyncNotifier<int?, UserIdFirestore?>
+    implements UserCentauriPointsViewModel {
+  final Future<int?> Function(UserIdFirestore? arg) _build;
+  final Future<void> Function() _onRefresh;
+  final VoidCallback _onRefreshWithCentauriPointsNumber;
+
+  MockUserAvatarCentauriPointsViewModel({
+    Future<int?> Function(UserIdFirestore? arg)? build,
+    Future<void> Function()? onRefresh,
+    VoidCallback? onRefreshWithCentauriPointsNumber,
+  })  : _build = build ?? ((_) async => null),
+        _onRefresh = onRefresh ?? (() async {}),
+        _onRefreshWithCentauriPointsNumber =
+            onRefreshWithCentauriPointsNumber ?? (() {});
+
+  @override
+  Future<int?> build(UserIdFirestore? arg) => _build(arg);
+
+  @override
+  Future<void> refresh() => _onRefresh();
+
+  @override
+  void refreshWithCentauriPointsNumber(int centauriPoints) =>
+      _onRefreshWithCentauriPointsNumber();
+}
+
+List<Override> mockUserCentauriPointsViewModelCentauriOverride({
```

Comment:
I think it is worth adding some documentation here

Reply:
You are right 💯 

Added in: [54b80e6](https://github.com/ProximaEPFL/proxima/pull/348/commits/54b80e62e6891282a6cd1a744d8a32d5e7c209a1)

Thank you !